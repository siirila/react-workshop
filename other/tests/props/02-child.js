import React from 'react'
import {render} from 'enzyme'
const Parent = require(`../../../src/props/${process.env.TEST_DIR ||
  'exercises'}/02-child`).default

test('Renders "Passed in a child string" in a div', () => {
  expect(render(<Parent>Passed in a child string</Parent>)).toMatchSnapshot()
})

test('Renders <span>Child element passed in</span> in a div', () => {
  expect(
    render(
      <Parent>
        <span>Child element passed in</span>
      </Parent>,
    ),
  ).toMatchSnapshot()
})
