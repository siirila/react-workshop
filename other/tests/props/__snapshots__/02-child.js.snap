// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "Passed in a child string" in a div 1`] = `
<div>
  Passed in a child string
</div>
`;

exports[`Renders <span>Child element passed in</span> in a div 1`] = `
<div>
  <span>
    Child element passed in
  </span>
</div>
`;
